pkgs/applications/networking/cluster/helm/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv, fetchurl, kubernetes }:
+let
+  arch = if stdenv.isLinux
+         then "linux-amd64"
+         else "darwin-amd64";
+  checksum = if stdenv.isLinux
+             then "dad3791fb07e6cf34f4cf611728cb8ae109a75234498a888529a68ac6923f200"
+             else "d27bd7e40e12c0a5f08782a8a883166008565b28e0b82126d2089300ff3f8465";
+in
+stdenv.mkDerivation rec {
+  pname = "helm";
+  version = "2.0.2";
+  name = "${pname}-${version}";
+
+  src = fetchurl {
+    url = "https://kubernetes-helm.storage.googleapis.com/helm-v${version}-${arch}.tar.gz";
+    sha256 = "${checksum}";
+  };
+
+  buildInputs = [ ];
+
+  propagatedBuildInputs = [ kubernetes ];
+
+  phases = [ "buildPhase" "installPhase" ];
+
+  buildPhase = ''
+    mkdir -p $out/bin
+  '';
+
+  installPhase = ''
+    tar -xvzf $src
+    cp ${arch}/helm $out/bin/${pname}
+    chmod +x $out/bin/${pname}
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/kubernetes/helm;
+    description = "A package manager for kubernetes";
+    license = licenses.asl20;
+    platforms = platforms.linux ++ platforms.darwin;
```

Comment:
Sure. There is a list of maintainers somewhere - presumably I need to add myself to that?